File: src/main/java/client/messages/IncMesoMessage.java
package client.messages;

import util.packet.PacketWriter;

public class IncMesoMessage extends AbstractMessage {

    private final int meso;

    public IncMesoMessage(int meso) {
        this.meso = meso;
    }

    @Override
    public MessageType getType() {
        return MessageType.INC_MESO_MESSAGE;
    }

    @Override
    protected void encodeData(PacketWriter pw) {
        pw.writeInt(meso);
    }
}

File: src/main/java/client/messages/IncMesoMessage.kt
package client.messages

import util.packet.PacketWriter

class IncMesoMessage(private val meso: Int) : AbstractMessage() {

    override val type: MessageType get() = MessageType.INC_MESO_MESSAGE

    override fun encodeData(pw: PacketWriter) {
        pw.writeInt(meso)
    }
}